drivers/led/lp5562.c

```diff
@@ -912,6 +940,11 @@ static int lp5562_led_init(const struct device *dev)
 	dev_data->min_brightness = LP5562_MIN_BRIGHTNESS;
 	dev_data->max_brightness = LP5562_MAX_BRIGHTNESS;
 
+	if (lp5562_led_update_current(dev)) {
+		LOG_ERR("Setting current setting LP5562 LED chip failed.");
+		return -EIO;
```

Comment:
It would be better to return the value returned by `i2c_write()` instead of `-EIO`. I know `-EIO` is the only error code returned but if that changes in the future then this code will be ready :)